default/stringtables/es.txt

```diff
@@ -7362,127 +7362,128 @@ El imperio %empire% ha sido eliminado.
+
+BEGINNER_HINT_16
+16: Puede que quieras retrasar la investigación de [[tech LRN_PSIONICS]] hasta que tu imperio incluya especies con [[encyclopedia TELEPATHIC_TITLE]].
+
+BEGINNER_HINT_17
+17: [[metertype METER_SHIELD]] provee defensa a una nave contra todo disparo contra la misma, reduciendo el daño recibido en el valor del escudo. Los valores de escudo de una nave pueden verse afectados por varios factores, como tormentas solares, en qué tipo de sistema estelar se encuentra la nave, o si estás cerca una capital imperial capturada.
+
+BEGINNER_HINT_18
+18: [[encyclopedia FOCUS_TITLE]] - presta atención al medidor de enfoque de los nuevos planetas que añadas a tu imperio, tanto colonizados como conquistados. Este medidor controla si el planeta se enfoca en la producción industrial, en investigación o en defensa. Avanzada la partida, más opciones de enfoque pueden ser desbloqueadas a medida que se investigan nuevas [[encyclopedia ENC_TECH]]s. Distintas especies  pueden tener disferentes opciones de enfoque disponibles.
+
+BEGINNER_HINT_19
+19: Se puede construir una colonia en un [[encyclopedia OUTPOSTS_TITLE]] construyendo un edificio colonial o mediante una [[predefinedshipdesign SD_COLONY_SHIP]]. La mayoría de edificios coloniales requieren una conexión de [[metertype METER_SUPPLY]] a un planeta con un medidor de [[metertype METER_HAPPINESS]] de al menos 5, y una [[metertype METER_POPULATION]] de 3. Cuando se desbloquea una especie, como al investigar [[tech PRO_EXOBOTS]], no se aplica este requisito para su edificio colonial.
+
+BEGINNER_HINT_20
+20: Informes de situación concretos (como este) pueden ser temporalmente ignorados haciendo click con el botón derecho del ratón en su icono y seleccionando una de las opciones disponibles. Ciertos tipos de reportes pueden ser filtrados de manera permanente usando el menú de filtros en la parte inferior de esta pantalla . Estas pistas iniciales están agrupadas como [[encyclopedia BEGINNER_HINTS]] en la lista de filtros.
+
+BEGINNER_HINT_21
+21: Puedes fijar un destino para una nave mientras está todavía en la cola de producción. Con la pantalla de producción abierta: haz click en el sistema destino en el mapa galáctico y haz click con el botón derecho del ratón sobre la nave en la cola de producción. Cuando aparezca el menú con las distintas opciones elige 'Ir a' y entonces debería aparecer el nombre del sistema seleccionado al que quieres que viaje la nave tan pronto como esté construida.
+
+BEGINNER_HINT_22
+22: Seleccionar un systema y pulsar ALT-C proporciona un círculo de rango para ese sistema, haz zoom para comparar la distancia mirando el tamaño del medidor de distancia.
```

Comment:
```suggestion
22: Seleccionar un sistema y pulsar ALT-C proporciona un círculo de rango para ese sistema, haz zoom para comparar la distancia mirando el tamaño de la barra de escala en la esquina superior izquierda del mapa.
```

I'm deliverately improving this hint for the Spanish version (the scale bar is not a distance meter indeed).